src/Business Foundation/App/NoSeries/src/Setup/NoSeriesLines.Page.al

```diff
@@ -0,0 +1,167 @@
+        {
+            repeater(Control1)
+            {
+                ShowCaption = false;
+                field("Series Code"; Rec."Series Code")
+                {
+                    Caption = 'Series Code';
+                    ToolTip = 'Specifies the code for the number series to which this line applies.';
+                    Visible = false;
+                }
+                field("Starting Date"; Rec."Starting Date")
+                {
+                    Caption = 'Starting Date';
+                    ToolTip = 'Specifies the date from which you would like this line to apply.';
+                }
+                field("Starting No."; Rec."Starting No.")
+                {
+                    Caption = 'Starting No.';
+                    ToolTip = 'Specifies the first number in the series.';
+                }
+                field("Ending No."; Rec."Ending No.")
+                {
+                    Caption = 'Ending No.';
+                    ToolTip = 'Specifies the last number in the series.';
+                }
+                field("Last Date Used"; Rec."Last Date Used")
+                {
+                    Caption = 'Last Date Used';
+                    ToolTip = 'Specifies the date when a number was most recently assigned from the number series.';
+                }
+                field("Last No. Used"; Rec."Last No. Used")
+                {
+                    Caption = 'Last No. Used';
+                    ToolTip = 'Specifies the last number that was used from the number series.';
+                    trigger OnValidate()
+                    begin
+                        CurrPage.Update(true);
+                    end;
+                }
+                field("Warning No."; Rec."Warning No.")
+                {
+                    Caption = 'Warning No.';
+                    ToolTip = 'Specifies when you want to receive a warning that the number series is running out.';
+                }
+                field("Increment-by No."; Rec."Increment-by No.")
+                {
+                    Caption = 'Increment-by No.';
+                    ToolTip = 'Specifies the size of the interval by which you would like to space the numbers in the number series.';
+                    trigger OnValidate()
+                    begin
+                        CurrPage.Update(true);
+                    end;
+                }
+                field("Allow Gaps in Nos."; AllowGaps)
+                {
+                    Caption = 'Allow Gaps in Nos.';
+                    ToolTip = 'Specifies that a number assigned from the number series can later be deleted. This is practical for records, such as item cards and warehouse documents that, unlike financial transactions, can be deleted and cause gaps in the number sequence. NOTE: If an error occurs on a new record that will be assigned a number from such a number series when it is completed, the number in question may be lost, causing a gap in the sequence.';
```

Comment:
The tooltip is very long and sounds more like a note that needs to be added into Docs rather than tooltip.

Reply:
This is an existing tooltip, leaving for now we have a slice to look into these things.